src/IdentityServer4.RavenDB.Storage/DocumentStoreHolder/OperationalDocumentStoreHolder.cs

```diff
@@ -1,14 +1,28 @@
 ﻿using System;
-using System.Runtime.CompilerServices;
+using IdentityServer4.RavenDB.Storage.Helpers;
+using IdentityServer4.RavenDB.Storage.Options;
 using Raven.Client.Documents;
+using Raven.Client.Documents.Session;
 
-[assembly: InternalsVisibleTo("IdentityServer4.RavenDB.IntegrationTests")]
 namespace IdentityServer4.RavenDB.Storage.DocumentStoreHolder
 {
-    internal class OperationalDocumentStoreHolder : DocumentStoreHolderBase, IDisposable
+    internal class OperationalDocumentStoreHolder : IDisposable

```

Comment:
Please consider introducing an interface that would connect both `ConfigurationDocumentStoreHolder` and `OperationalDocumentStoreHolder`.

Reply:
The two stores need to be treated separately as they are functionally independent of each other.  The `OperationalStore` does not require the `ConfigurationalStore` to be added and vice versa. That is why I gave up the idea of introducing a base class. However, an interface is a lightweight type so we might give it a try to show that as far as database connection management goes the two stores are conceptually similar.